chapter2.adoc

```diff
@@ -79,7 +79,12 @@ The dcsr (at 0x7b0) is always accessible in Debug Mode and the `prv` and `v` fie
 
 [NOTE]
 As the `prv` and `v` fields in dcsr are Write Any Read Legal (WARL) fields, the external debugger is able to read back the written value to determine the maximum debug privilege level.  
- 
+
+[[phyaddr]]
```

Comment:
merge with ammvirtual?

Reply:
I am going to change it as follows:

> The field aamvirtual in the command (at 0x17 in the Debug Module) determines whether the Access Memory command uses a physical or virtual address. When an Access Memory command is issued with aamvirtual=0, the hart must check whether the physical access is allowed to access memory. The hart responds with an exception to the Debug Module when M-mode is not permitted to debug, tvm (in mstatus) is set to 1, and mode (in satp) enables any kind of virtual translation. In the event of an exception, the Debug Module set cmderr to 3 and clear the data registers to 0.

What do you think of it?

Reply:
LGTM